drools-osgi/drools-karaf-itests/src/test/java/org/drools/karaf/itest/util/PaxExamWithWireMock.java

```diff
@@ -0,0 +1,665 @@
+package org.drools.karaf.itest.util;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.net.ServerSocket;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.model.InitializationError;
+import org.ops4j.pax.exam.junit.PaxExam;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.*;
+
+public class PaxExamWithWireMock extends PaxExam {
+
```

Comment:
Maybe we could use `PaxExamParameterized` it basically combines PaxExam with default jUnit's Parametrized runner. We could hook the wiremock as the tests are usually the same for XML and JSON?